tests/service/Symbols.fs

```diff
@@ -474,3 +474,48 @@ type Foo =
           (:? FSharpMemberOrFunctionOrValue as setMfv) ->
             Assert.AreNotEqual(getMfv.CurriedParameterGroups, setMfv.CurriedParameterGroups)
         | _ -> Assert.Fail "Expected symbols to be FSharpMemberOrFunctionOrValue"
+        
+    [<Test>]
```

Comment:
This is not complete @vzarytovskii.
`module GetValSignatureText =` is still missing.
https://github.com/dotnet/fsharp/pull/15625/files#diff-86de212e8dc5b638058239f2b2b485cd6207f61b3acb2556de8bb7fef15065eaL619-L680

Reply:
One test is also failing - 
https://dev.azure.com/dnceng-public/public/_build/results?buildId=344741&view=ms.vss-test-web.build-test-results-tab&runId=7207146&resultId=100376&paneView=debug